src/components/AppStructure.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PureComponent from 'react-pure-render/component';
+import Header from './Header.jsx';
+import Footer from './Footer.jsx';
```

Comment:
Nit: Move this import above the `Header.jsx` import, since F comes before H alphabetically.